migrations/2018-04-17-subscribers-table.sql

```diff
@@ -0,0 +1,12 @@
+-- This file creates a new table, subscriptions, in the ejabberd DB.
+
+USE ejabberd;
+
+CREATE TABLE subscriptions (
+    name text NOT NULL,
```

Comment:
generally best practice to define varchars with fixed lengths (if you know what the max size could be).

Reply:
Will change to varchar(50) since its the concatenation of 2 ids.

Reply:
Should I change the other text as well?